laws/src/main/scala/cats/laws/discipline/ArbitraryK.scala

```diff
@@ -41,4 +42,10 @@ object ArbitraryK {
 
   implicit def cokleisliA[F[_], A]: ArbitraryK[Cokleisli[F, A, ?]] =
     new ArbitraryK[Cokleisli[F, A, ?]]{ def synthesize[B: Arbitrary]: Arbitrary[Cokleisli[F, A, B]] = implicitly }
+
+  implicit def futureArbitraryK(implicit ec: ExecutionContext): ArbitraryK[Future] =
```

Comment:
Can we drop this implicit `ExecutionContext` since we are just using `Future.successful`?